shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml

```diff
@@ -510,4 +510,13 @@
     <test-case sql="SELECT * FROM t_user u INNER JOIN t_user_item m ON u.user_id=m.user_id" scenario-types="encrypt,dbtbl_with_read_write_splitting_and_encrypt">
         <assertion expected-data-file="select_join_encrypt.xml"/>
     </test-case>
+
+    <test-case sql="SELECT * FROM t_single_table" db-types="MySQL,Oracle,SQLServer" scenario-types="db">
+        <assertion expected-data-file="select_from_single.xml"/>
+    </test-case>
+
+    <!--fixme! issue #9719-->
```

Comment:
IMO, you could fix the issue #9719 in this pull request, not adding the comments here to fix.

Reply:
> IMO, you could fix the issue #9719 in this pull request, not adding the comments here to fix.

In fact, many of our cases are annotated in this way. This may make it easier for us to review the problem.

Reply:
@JiekerTime IMO, it's better to separated into new commit or new PR for this change.
@tristaZero what do you think about?

Reply:
@SteNicholas , This issue will be handled by @JiekerTime soon and this PR firstly prepare the unit cases.